src/python/WMCore/WorkQueue/WMBSHelper.py

```diff
@@ -634,9 +634,13 @@ def _addDBSFileToWMBSFile(self, dbsFile, storageElements, inFileset=True):
 
         for lumi in dbsFile['LumiList']:
             if isinstance(lumi['LumiSectionNumber'], list):
-                run = Run(lumi['RunNumber'], *lumi['LumiSectionNumber'])
+                lumiSecList = (list(zip(lumi['LumiSectionNumber'], lumi['EventCount']))
```

Comment:
ACDC format is different from DBS.
DBS has 2 arrays one for lumi and the other for events. ACDC I put it as list of tuples [(lumi, events), ...] since it matches better with WMCore run structure.